babel.cfg

```diff
@@ -0,0 +1,6 @@
+# See https://github.com/python-babel/babel/blob/2a1709a7768f6f07c3d2dbfdb03d3c8a6bd80aef/docs/messages.rst#extraction-method-mapping-and-configuration
+[python: **.py]
+[jinja2: **.html]
+encoding = utf-8
+ignore_tags = script,style
+include_attrs = alt title summary placeholder
```

Comment:
Do we need these lines for Jinja2, or are they just applicable to Genshi?

Reply:
I think they are unneeded, have removed